sdk/search/search-documents/src/searchClient.ts

```diff
@@ -584,12 +594,28 @@ export class SearchClient<T> {
   public async deleteDocuments(
     keyName: keyof T,
     keyValues: string[],
+    options?: DeleteDocumentsOptions
+  ): Promise<IndexDocumentsResult>;
+
+  /**
+   * Delete a set of documents by their primary key.
```

Comment:
Again, we shouldn't doc comment the implementation since we want users to use the other two signature forms instead

Reply:
Done